Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/ScaleHandles.cs

```diff
@@ -63,44 +70,116 @@ internal void Create(ref Vector3[] boundsCorners, Transform parent, bool drawMan
                     visualsScale.transform.localScale = new Vector3(Mathf.Sign(p[0]), Mathf.Sign(p[1]), Mathf.Sign(p[2]));
                 }
 
-                // figure out which prefab to instantiate
-                GameObject cornerVisual = null;
-                GameObject prefabType = isFlattened ? config.HandleSlatePrefab : config.HandlePrefab;
-                if (prefabType == null)
-                {
-                    // instantiate default prefab, a cube. Remove the box collider from it
-                    cornerVisual = GameObject.CreatePrimitive(PrimitiveType.Cube);
-                    cornerVisual.transform.parent = visualsScale.transform;
-                    cornerVisual.transform.localPosition = Vector3.zero;
-                    GameObject.Destroy(cornerVisual.GetComponent<BoxCollider>());
-                }
-                else
-                {
-                    cornerVisual = GameObject.Instantiate(prefabType, visualsScale.transform);
-                }
+                Bounds visualBounds = CreateVisual(visualsScale, isFlattened);
+
+                var invScale = config.HandleSize / visualBounds.size.x;
```

Comment:
warning: possible divide by zero here. consider either adding small value to the bottom, or checking for zero?